state-state.js

```diff
@@ -43,4 +45,54 @@ module.exports = function StateState() {
 		getParent: getParent,
 		getParentName: getParentName
 	}
+
```

Comment:
It seems a little weird that some properties are added in the object literal, and some are added manually.

Is there a reason for this?